src/common/opencl.c

```diff
@@ -1152,57 +1154,52 @@ void dt_opencl_init(
 
   if(exclude_opencl)
   {
-    dt_print_nts(DT_DEBUG_OPENCL,
+    dt_print(DT_DEBUG_OPENCL,
                  "[opencl_init] opencl disabled due to explicit user request\n");
     goto finally;
   }
 
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl related configuration options:\n");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl: %s\n", opencl_requested ? "ON" : "OFF" );
-  const char *str = dt_conf_get_string_const("opencl_scheduling_profile");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_scheduling_profile: '%s'\n", str);
+  if(!opencl_requested)
+    dt_print(DT_DEBUG_OPENCL, "[opencl_init] opencl disabled via darktable preferences\n");
+
   // look for explicit definition of opencl_runtime library in preferences
   const char *library = dt_conf_get_string_const("opencl_library");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_library: '%s'\n",
-               (strlen(library) == 0) ? "default path" : library);
-  str = dt_conf_get_string_const("opencl_device_priority");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_device_priority: '%s'\n", str);
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_mandatory_timeout: %d\n",
-               dt_conf_get_int("opencl_mandatory_timeout"));
 
   // dynamically load opencl runtime and bind required symbols and test it
   cl->dlocl = dt_dlopencl_init(library);
   if(cl->dlocl == NULL)
   {
     logerror = "no working opencl library found";
-    dt_print_nts(DT_DEBUG_OPENCL,
-                 "[opencl_init] no working opencl library found."
-                 " Continue with opencl disabled\n");
+    dt_print(DT_DEBUG_OPENCL,
+                 "[opencl_init] no working opencl '%s' library found."
+                 " Continue with opencl disabled\n",
+                 (strlen(library) == 0) ? "default path" : library);
     goto finally;
   }
   else
   {
-    dt_print_nts(DT_DEBUG_OPENCL,
-                 "[opencl_init] opencl library '%s' found on your system and loaded\n",
-                 cl->dlocl->library);
+    dt_print(DT_DEBUG_OPENCL,
+                 "[opencl_init] opencl library '%s' found on your system and loaded, preference '%s'\n",
+                 cl->dlocl->library,
+                 (strlen(library) == 0) ? "default path" : library);
   }
 
   all_platforms = malloc(sizeof(cl_platform_id) * DT_OPENCL_MAX_PLATFORMS);
   all_num_devices = malloc(sizeof(cl_uint) * DT_OPENCL_MAX_PLATFORMS);
 
   cl_uint num_platforms = 0;
+  logerror =  "platform detection failed. Some possible reasons:\n"
```

Comment:
No capital letters :) So Some -> some / Previous -> previous

Reply:
For the list of items itself, add a `,` on each line except the last one where you want to add a `.`.

Reply:
done